packages/vscode-extension/lib/wrapper.js

```diff
@@ -69,6 +69,118 @@ function useAgentListener(agent, eventName, listener, deps = []) {
   }, [agent, ...deps]);
 }
 
+function getInspectorDataForInstance(node) {
+  const renderers = Array.from(window.__REACT_DEVTOOLS_GLOBAL_HOOK__?.renderers?.values());
+  if (!renderers) {
+    return {};
+  }
+  for (const renderer of renderers) {
+    if (renderer.rendererConfig?.getInspectorDataForInstance) {
+      const data = renderer.rendererConfig.getInspectorDataForInstance(node);
```

Comment:
we will find the exact same `rendererConfig` object at every iteration of the loop from traversal. It'd seem to make more sense to do it once and not separately for every single node

Reply:
as above